social/src/routes/twitter/home-timeline.ts

```diff
@@ -0,0 +1,48 @@
+import express, { Request, Response } from 'express';
+import {
+  BadRequestError,
+  FailedConnectionError,
+  ForbiddenError,
+  requireAuth,
+} from '@tcosmin/common';
```

Comment:
How did you end up choosing this library? Looks like it is a less used one, usually I would suggest look for the most used ones.

Reply:
This is my own library. I use this to share Errors, Events and some Middlewares between microservices

Reply:
Well done improving the community with your own libraries!